docs/usage.rst

```diff
@@ -424,16 +431,34 @@ It is possible to extract BIN files from a device via USB. To convert these bina
 
 Event markers
 """""""""""""
-
-In addition to the marker event generated by pressing the button on your Explore device, you can set markers programmatically using:
-::
-    explorepy.Explore.set_marker
-
-However, this function must be called from its own, dedicated thread.
-
-.. note:: Marker codes 0-7 are reserved for the hardware. You can use any other (integer) code for your marker (8-65535).
-
-For an example, see `this script <https://github.com/Mentalab-hub/explorepy/tree/master/examples/marker_example.py>`_.
+Event markers can be used for time synchronization in the analyses. The following table describes all type of event markers
```

Comment:
"Event markers can be used for time synchronization in the analyses." ->

"Event markers can be used to time synch data."